galaxy/importer/loaders/role.py

```diff
@@ -274,7 +274,17 @@ def _load_metadata(self):
         else:
             self.log.warning("Missing 'galaxy_info' field in metadata.")
             galaxy_info = {}
-        return galaxy_info
+
+        if 'dependencies' in metadata:
```

Comment:
Probably it's better to move all this logic to RoleMetaParser to follow single responsibility principle.

Reply:
Agreed. Moved.